ansible/roles/jenkins/tasks/main.yml

```diff
@@ -50,10 +50,22 @@
     -p NOTIFY_EMAIL_LIST={{ notify_email_list }} | {{ oc }} apply --force=true -f - -n {{ project_name }}"
 - name: Create Jenkins master pipeline
   shell: "{{ oc }} process -f {{ role_path }}/files/jenkins-pipeline.yaml | {{ oc }} apply --force=true -f -"
-- name: Create application base image pipeline
-  shell: "{{ oc }} process -f {{ role_path }}/files/base-image-pipeline.yaml --param BASE_IMAGE_TAG='nodejs:6' --param APP_BC='nodejs-mongo-persistent' --param APP_NAME='nodejs-mongo-persistent' --param EMAIL_LIST='aweiteka@redhat.com jcallen@redhat.com' --param=APP_PIPELINE='lifecycle-app-lifecycle' | {{ oc }} apply --force=true -f -"
 - name: Create Jenkins base image pipeline
-  shell: "{{ oc }} process -f {{ role_path }}/files/base-image-pipeline.yaml | {{ oc }} apply --force=true -f -"
+  # Most of the defaults in the template are already set to match Jenkins
+  shell: >-
+    {{ oc }} process -f {{ role_path }}/files/base-image-pipeline.yaml
+    --param APP_NAMESPACE={{ project_name }}
+    | {{ oc }} apply --force=true -f -
+- name: Create application base image pipeline
+  # Using the same template but changing some parmeters to match the app
+  shell: >-
+    {{ oc }} process -f {{ role_path }}/files/base-image-pipeline.yaml
+    --param BASE_IMAGE_TAG='nodejs:6'
+    --param APP_BC='nodejs-mongo-persistent'
+    --param APP_NAME='nodejs-mongo-persistent'
+    --param APP_PIPELINE='lifecycle-app-lifecycle'
```

Comment:
@aweiteka is that right name of the pipeline?  https://github.com/RHsyseng/jenkins-on-openshift/blob/f841e6bb4743e4dde4b70d9bdf3a1159eea381a6/ansible/roles/jenkins/files/app-pipeline.yaml#L10

Reply:
Oh wait this is the name in Jenkins.  But isn't it a slash vs a hyphen?

Reply:
nevermind, display name vs project name.